target/linux/ramips/mt7620/profiles/zbtlink.mk

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
OK, sure, we don't need USB at all.
Selecting non-defaul profile saves flash space;-)
Is it OK?